lisp/ess-stata-mode.el

```diff
@@ -83,7 +83,6 @@
     (inferior-ess-primary-prompt   . "[.:] \\|--more--") 
     (inferior-ess-secondary-prompt . "--more--")
     (comint-use-prompt-regexp      . t)
-    (inferior-ess-start-args       . inferior-STA-start-args)
```

Comment:
Doesn't inferior-STA-start-args neeed to be used somewhere?

Reply:
This definition in the cust-alist was redundant, `inferior-STA-starts-args` is already being passed to `inferior-ess`. This shows how this variable was confusing and it's a good thing to obsolete it.

Reply:
Ah, I see that now. Thanks!